src/relay/ir/dataflow_matcher.cc

```diff
@@ -796,24 +796,38 @@ Expr PatternRewriter::Rewrite(const Array<DFPatternCallback>& callbacks, const E
   bool equal = true;
   static auto* structural_equal = runtime::Registry::Get("node.StructuralEqual");
   ICHECK(structural_equal) << "node.StructuralEqual is not registered.";
+  // Use the callback until the callback's attribute is rewrite_once=true and has been rewritten
+  std::unordered_map<DFPatternCallback, bool, ObjectPtrHash, ObjectPtrEqual> callbacks_map;
```

Comment:
`callbacks_map` -> `done`, swapping false and true

Reply:
Please take another look. Thank you!